spec/components/previews/layouts/component_preview.html.erb

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html style="background-color: <%= params[:lookbook][:display][:bg_color] || "white" %>">
+  <head>
+
+    <title>
+      Component Previews
+    </title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <%= stylesheet_link_tag 'application_stylesheet', media: 'all'  %>
+    <%= stylesheet_pack_tag 'application' %>
+
+    <%= javascript_pack_tag 'application', 'data-turbo-track': 'reload', defer: true %>
+  </head>
+
+  <body style="padding: 20px">
+  <div style="
+    margin-left: auto;
+    margin-right: auto;
+    max-width: <%= params[:lookbook][:display][:max_width] || "100%" %>
+  ">
+    <%= yield %>
+  </div>
+</html>
```

Comment:
Ended up using the boilerplate example from the lookbook docs - could be improved, but simple and does the job for now
https://lookbook.build/guide/previews/layouts/